### ISSUE TITLE
og:image is not working on Twitter

### ISSUE THREAD

**Issue by mushon** [akariv/gov-checks#5]

**Comment by akariv** [comment]
Seems to be working for me...

<img width="671" alt="Screen Shot 2023-06-19 at 9 44 46" src="https://github.com/akariv/gov-checks/assets/265333/4c1148fe-9e2f-44fb-81cc-dd1557045664">

**Comment by mushon** [comment]
True, but not in [the English version](https://protect-democracy.berl.org.il/en/ ): 
<img width="586" alt="image" src="https://github.com/akariv/gov-checks/assets/27123/38fa164b-890f-4ed1-8df5-e62683c67c89">

**Comment by akariv** [comment]
Looks like a Twitter cache issue.
<img width="605" alt="Screen Shot 2023-06-19 at 10 13 50" src="https://github.com/akariv/gov-checks/assets/265333/f815d9e6-725b-4f83-85ed-5ad838d1358e">

They say they re-crawl all pages every week:

<img width="1001" alt="Screen Shot 2023-06-19 at 10 11 41" src="https://github.com/akariv/gov-checks/assets/265333/6010d501-8900-46be-a3a2-f00c68b96ce0">
